sdk/eventhub/event-hubs/test/iothub.spec.ts

```diff
@@ -29,13 +29,22 @@ describe("EventHub Client with iothub connection string", function(): void {
   });
 
   it("should be able to get hub runtime info", async function(): Promise<void> {
+
     client = await EventHubClient.createFromIotHubConnectionString(service.connectionString!);
     const runtimeInfo = await client.getHubRuntimeInformation();
     debug(">>> RuntimeInfo: ", runtimeInfo);
-    should.exist(runtimeInfo);
-    runtimeInfo.type.should.equal("com.microsoft:eventhub");
-    runtimeInfo.partitionCount.should.be.greaterThan(0);
-    runtimeInfo.partitionIds.length.should.be.greaterThan(0);
+    should.exist(runtimeInfo, `RuntimeIno does not exist. Found ${runtimeInfo}`);
+    should.equal(runtimeInfo.type, "com.microsoft:eventhub", `Runtime Type is not equal and found ${runtimeInfo.type}`);
```

Comment:
What is the reason to change the way `should` is used? Did it cause troubles for browserification?

Reply:
Yes, there were some conflicts. Other files were using should differently and had no issue, hence made them consistent.